src/main/flight/pid.c

```diff
@@ -152,7 +152,7 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .horizon_tilt_effect = 75,
         .horizon_tilt_expert_mode = false,
         .crash_limit_yaw = 200,
-        .itermLimit = 150,
+        .itermLimit = 300,
```

Comment:
Too high iterm limit can cause quads to take off by itself in some cases! In this case you give ~30% throttle to iterm!

Let's say someone wants to take off, but grass is in the way and quad can't take off quickly than iterm accumulation could cause too much power going into motors.

Reply:
Did you see iterm going above this threshold in any of the logs btw?

Reply:
There is a protection against I windup after arming, I think, unless throttle is increased above air mode threshold.
Yes for yaw it is easy for I to go this high.  I do not see it on pitch and roll.  It might be better to have separate limits for I on yaw vs pitch and roll